app/scripts/modules/core/src/projects/ProjectSearchResultFormatter.ts

```diff
@@ -16,13 +14,18 @@ export class ProjectSearchResultFormatter implements ISearchResultFormatter {
 
   public get displayName() { return 'Projects'; }
   public get order() { return 0; }
+  public get icon() { return 'hand-spock-o'; }
```

Comment:
We need to decide on a proper Project icon....as cool as _live long and prosper_ is...

Reply:
It's not.